cluster-api/src/main/java/io/aiven/klaw/clusterapi/controller/ConsumerGroupsController.java

```diff
@@ -62,7 +61,6 @@ public ApiResponse resetConsumerOffsets(
   @RequestMapping(
       value =
           "/getConsumerOffsets/{bootstrapServers}/{protocol}/{clusterName}/{consumerGroupId}/{topicName}",
-      method = RequestMethod.GET,
       produces = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
Sorry I meant here you could change it to be like 
@RequestMapping(
      value =
          "/consumerOffsets/{bootstrapServers}/{protocol}/{clusterName}/{consumerGroupId}/{topicName}",
      method = RequestMethod.GET,
      produces = {MediaType.APPLICATION_JSON_VALUE})
      
      I think we can drop the get at the start of getConsumerOffsets/....

Reply:
Got it, updated.

Reply:
Thanks Murali, does it require "method = RequestMethod.GET," to be added back?